sbin/wifibox

```diff
@@ -734,9 +739,28 @@ vm_manager() {
 
     log debug "Nice priority: ${_nice_priority}"
     log debug "Arguments: ${_bhyve_args}"
-    ${NICE} -n ${_nice_priority} \
-	    ${BHYVE} ${_bhyve_args} 2>&1 | capture_output debug bhyve
+
+    _kill_child() {
+	local _nice_pid
+	_nice_pid="$(jobs -p)"
```

Comment:
Semantic nitpicking: the invocation `jobs` suggests that there would be more process to return other than `nice`.  It is clearly a list of processes.  We should either take the first element of this list or grab all the returned values at once and `kill` them all.  I think the first option is brittle therefore the second one would be a better choice.